### PR TITLE
Option for not downloading runtime deployment files in Arktos onebox setup

### PR DESCRIPTION
**Pull request by yb01**
**What type of PR is this?**
> /kind bug fix

**What this PR does / why we need it**:
Fix issue Arktos issue 166. provides an option for not downloading runtime deployment files from runtime repo if a local copy is prefered to use

**Which issue(s) this PR fixes**:
Fixes 166

Tested locally:

1. Verify normal start of onebox will download runtime deployment files
  ./hack/arktos-up.sh 
2. with files in local directory, verify flag=false won't download them
  OVERWRITE_DEPLOYMENT_FILES=false ./hack/arktos-up.sh 
3. remove the local files and retry to ensure files are downloaded, even flag=false
  rm /tmp/* -f -r
  OVERWRITE_DEPLOYMENT_FILES=false ./hack/arktos-up.sh

### CODE REVIEW

File: hack/lib/common.sh [hunk 1]
```diff
@@ -57,12 +57,29 @@ fi
...
+  wget -O ${VIRTLET_DEPLOYMENT_FILES_DIR}/vms  ${VIRTLET_DEPLOYMENT_FILES_SRC}/apparmor/vms
+fi
+
+if [[ (${OVERWRITE_DEPLOYMENT_FILES} == "true") || (! -f ${VIRTLET_DEPLOYMENT_FILES_DIR}/vmruntime.yaml) ]]; then
+  wget -O ${VIRTLET_DEPLOYMENT_FILES_DIR}/vmruntime.yaml  ${VIRTLET_DEPLOYMENT_FILES_SRC}/data/virtlet-ds.yaml
+fi
+
+if [[ (${OVERWRITE_DEPLOYMENT_FILES} == "true") || (! -f ${VIRTLET_DEPLOYMENT_FILES_DIR}/images.yaml) ]]; then
+  wget -O ${VIRTLET_DEPLOYMENT_FILES_DIR}/images.yaml  ${VIRTLET_DEPLOYMENT_FILES_SRC}/images.yaml
+fi
```
Comment (pdgetrf) [inline hack/lib/common.sh:82]: maybe put these into a function to prevent repeating it

**Comment by pdgetrf** [comment]
Need a bit more info in the description on how this works and how this was tested. Something like "setting the env variable xxx will prevent downloading from internet".

File: hack/lib/common.sh [hunk 2]
```diff
@@ -57,12 +57,29 @@ fi
...
+if [[ (${OVERWRITE_DEPLOYMENT_FILES} == "true") || (! -f ${VIRTLET_DEPLOYMENT_FILES_DIR}/vms) ]]; then
+  wget -O ${VIRTLET_DEPLOYMENT_FILES_DIR}/vms  ${VIRTLET_DEPLOYMENT_FILES_SRC}/apparmor/vms
+fi
+
+if [[ (${OVERWRITE_DEPLOYMENT_FILES} == "true") || (! -f ${VIRTLET_DEPLOYMENT_FILES_DIR}/vmruntime.yaml) ]]; then
+  wget -O ${VIRTLET_DEPLOYMENT_FILES_DIR}/vmruntime.yaml  ${VIRTLET_DEPLOYMENT_FILES_SRC}/data/virtlet-ds.yaml
+fi
+
+if [[ (${OVERWRITE_DEPLOYMENT_FILES} == "true") || (! -f ${VIRTLET_DEPLOYMENT_FILES_DIR}/images.yaml) ]]; then
+  wget -O ${VIRTLET_DEPLOYMENT_FILES_DIR}/images.yaml  ${VIRTLET_DEPLOYMENT_FILES_SRC}/images.yaml
```
Comment (pdgetrf) [inline hack/lib/common.sh:81]: will "make clean" remove the previously downloaded local copy?

Comment (yb01) [reply hack/lib/common.sh:81]: no, make clean won't delete them. they can be put into user specified folder to keep them for future uses.

File: hack/lib/common.sh (hunk 1 above)
Comment (yb01) [reply hack/lib/common.sh:82]: good point. will do so.

**Comment by yb01** [comment]
update the PR description with local test cases

**Pull request merged**